File: primer_trimestre/poo/php/ejercicios_php/first/libreria/ExtrasHours.php
<?php
class ExtrasHours {
    private $dHours;
    private $nHours;
    private $hHours;

    public function __construct($dHours, $nHours, $hHours) {
        $this->dHours = $dHours;
        $this->nHours = $nHours;
        $this->hHours = $hHours;
    }

    public function getDHoras() {
        return $this->dHours;
    }

    public function getNHoras() {
        return $this->nHours;
    }

    public function getHHoras() {
        return $this->hHours;
    }

    public function calculateEH() {
        return $this->dHours * 5000 * 1.25 + $this->nHours * 5000 * 1.75 + $this->hHours * 5000 * 2;
    }
}
?>

File: primer_trimestre/poo/java/proyectos_java/ejercicio_1_java/src/extrasHours.java
public class extrasHours {
    private int dHours;
    private int nHours;
    private int hHours;

    public extrasHours( int dHours, int nHours, int hHours) {
        this.dHours = dHours;
        this.nHours = nHours;
        this.hHours = hHours;
    }
    public double getdHours() {
        return this.dHours;
    }
    public double getnHours() {
        return this.nHours;
    }
    public double gethHours() {
        return this.hHours;
    }
    public double  calculateEH(){
        return this.dHours * 5000 * 1.25 + this.nHours * 5000 * 1.75 + this.hHours * 5000 * 2;
    }

}
